os/booting-with-libvirt.md

```diff
@@ -57,172 +62,240 @@ wget https://stable.release.core-os.net/amd64-usr/current/coreos_production_qemu
 Now create a qcow2 image snapshot using the command below:
 
 ```sh
-cd /var/lib/libvirt/images/coreos
-qemu-img create -f qcow2 -b coreos_production_qemu_image.img coreos1.qcow2
+cd /var/lib/libvirt/images/container-linux
+qemu-img create -f qcow2 -b coreos_production_qemu_image.img container-linux1.qcow2
 ```
 
-It will create `coreos1.qcow2` snapshot image. Any changes to `coreos1.qcow2` will not be reflected in `coreos_production_qemu_image.img`. Making any changes to a base image (`coreos_production_qemu_image.img` in our example) will corrupt its snapshots.
+This will create a `container-linux1.qcow2` snapshot image. Any changes to `container-linux1.qcow2` will not be reflected in `coreos_production_qemu_image.img`. Making any changes to a base image (`coreos_production_qemu_image.img` in our example) will corrupt its snapshots.
 
-### Config drive
+### Ignition config
 
-Now create a config drive file system to configure Container Linux itself:
+The preferred way to configure a Container Linux machine is via Ignition.
+Unfortunately, libvirt does not have direct support for Ignition yet, so configuring it involves including qemu-specific xml.
+
+This configuration can be done in the following steps:
+
+#### Create the Igntion config
+
+Typically you won't write Ignition files yourself, rather you will typically use a tool like the [config transpiler](https://coreos.com/os/docs/latest/overview-of-ct.html) to generate them.
+
+However the Ignition file is created, it should be placed in a location which qemu can access. In this example, we'll place it in `/var/lib/libvirt/container-linux/container-linux1/provision.ign`.
 
 ```sh
-mkdir -p /var/lib/libvirt/images/coreos/coreos1/openstack/latest
-touch /var/lib/libvirt/images/coreos/coreos1/openstack/latest/user_data
+mkdir -p /var/lib/libvirt/container-linux/container-linux1/
+echo '{"ignition":{"version":"2.0.0"}}' > /var/lib/libvirt/container-linux/container-linux1/provision.ign
 ```
 
 If the host uses SELinux, allow the VM access to the config:
 
 ```sh
-semanage fcontext -a -t virt_content_t "/var/lib/libvirt/images/coreos0/configdrive(/.*)?"
-restorecon -R "/var/lib/libvirt/images/coreos0/configdrive"
+semanage fcontext -a -t virt_content_t "/var/lib/libvirt/container-linux/container-linux1"
+restorecon -R "/var/lib/libvirt/container-linux/container-linux1"
 ```
 
-The `user_data` file declares machine configuration in the [cloud config](https://coreos.com/os/docs/latest/cloud-config.html) format. We recommend using ssh keys to log into the VM, and since those keys are stored in `user_data,` at minimum that file should contain something like this:
-
-```cloud-config
-#cloud-config
-
-ssh_authorized_keys:
- - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDGdByTgSVHq.......
+A simple Container Linux config to add your ssh keys might look like the following:
+
+```container-linux-config
```

Comment:
If you use `yaml container-linux-config`, it will render nicely on GitHub and the website.